tools/sof-dump-status.py

```diff
@@ -340,6 +341,55 @@ def _covertType(type_lst):
         print(card_info['longname'])
         exit(0)
 
+    # The kernel has changed the default firmware path when community
+    # key is used. Here we output firmware path according to kernel's
+    # match table.
+    if ret_args.get('fwpath') is True:
+        def is_community_board(board, community_boards):
+            for elem in community_boards:
+                if match_board(elem, board):
+                    return True
+            return False
+
+        def match_board(match, board):
+            for key in match["match"].keys():
+                if match["match"][key] == "":
+                    continue
+                else:
+                    if match["match"][key] != board["match"][key]:
+                        return False
+            return True
+
+        community_boards = [
+            {
+                "name": "Up Squared",
+                "match": {
+                    "board_name": "UP-APL01",
+                    "board_vendor": "AAEON"
+                }
+            },
+            {
+                "name": "Chromebook",
+                "match":  {
+                    "board_name": "",
```

Comment:
@aiChaoSONG Can we keep this filled not defined here? Then in the above check can we handle that there is no define in structure? This can save us some time if we need to add more DMI info here.

Reply:
@xiulipan  "board_name" field removed for Chromebook, for it make no sense on classify boards type. we only use the identification fields to classify a board, eg: For google, "vendor_name", and for Up2: "board_name" and "vendor_name".

Thank you for showing me ideas on implementation of such a little feature, illuminating me a lot.